subsys/bluetooth/audio/mcs.c

```diff
@@ -124,10 +125,14 @@ static ssize_t read_icon_id(struct bt_conn *conn,
 			    uint16_t len, uint16_t offset)
 {
 	uint64_t icon_id = media_proxy_sctrl_get_icon_id();
+	uint8_t icon_id_le[BT_OTS_OBJ_ID_SIZE];
+
+	sys_put_le48(icon_id, icon_id_le);
```

Comment:
An alternative to this could also be

```c
const uint8_t icon_id_le[BT_OTS_OBJ_ID_SIZE] = {BT_BYTES_LIST_LE48(icon_id)};
```

But the current solution is also perfectly fine